install/upgrade/versions/1.9.0_extras/migrate_SetHandler_to_ProxySetMatch.php

```diff
@@ -0,0 +1,159 @@
+<?php
```

Comment:
I think we allready rebuild all websites allready for 1.9 so those files will get replaced in the next step after this step is completed

https://github.com/hestiacp/hestiacp/blob/8d5bd57c8fc5756a8664d15f9a69478df413b45c/install/upgrade/versions/1.9.0.sh#L23

Reply:
oh dang. that's was a lot of wasted effort 😅

ok removed.